lms/djangoapps/lti_provider/views.py

```diff
@@ -149,4 +154,26 @@ def render_courseware():
     :return: an HttpResponse object that contains the template and necessary
     context to render the courseware.
     """
-    return HttpResponse('TODO: Render refactored courseware view.')
+    usage_id = lti_params['usage_id']
+    course_id = lti_params['course_id']
+    course_key = CourseKey.from_string(course_id)
+    user = request.user
+    course = get_course_with_access(user, 'load', course_key)
+    staff = has_access(request.user, 'staff', course)
+    instance, _ = get_module_by_usage_id(request, course_id, usage_id)
+
+    fragment = instance.render('student_view', context=request.GET)
+
+    context = {
+        'fragment': fragment,
+        'course': course,
+        'disable_accordion': True,
+        'allow_iframing': True,
+        'disable_header': True,
+        'disable_footer': True,
+        'disable_tabs': True,
+        'staff_access': staff,
+        'xqa_server': settings.FEATURES.get('USE_XQA_SERVER', 'http://xqa:server@content-qa.mitx.mit.edu/xqa'),
```

Comment:
I think the XQA server is not really much used anymore, either -- it was an experimental feature added in mid-2012 and I don't think maintained since then. I would much prefer to add some sort of setting to determine this url.

@carsongee could I get your input here -- what's the story behind the xqa server? Do you guys actively use/maintain this server? If it's still in use, would it be possible to have someone factor out all references to the server instead of having the url hardcoded in our codebase?